packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -179,6 +180,111 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * generate a deterministic HMAC for a reaction
+   * @param {Object} displayName displayName of reaction we are sending
+   * @param {Object} parent parent activity of reaction
+   * @returns {Promise<HMAC>}
```

Comment:
Can you mark this as `@private` so we don't generate docs for it publicly since this is a helper function?